src/routes_admin.py

```diff
@@ -30,9 +30,9 @@ def get_course_roster_page(netid, cid):
         def get_course_staff_roster(netid, cid):
             course = db.get_course(cid)
             
-            staff = course['staff']
+            staff = course["staff"]
             # get all admin_ids by filtering out all non-admins
-            admin = dict(filter(lambda x : x[1] == {'is_admin': True}, staff.items()))
+            admin = dict(filter(lambda x : x[1].get("is_admin", True), staff.items()))
```

Comment:
The second parameter of `dict.get` is the default value returned when there is not such key as `is_admin`. So in this code, if a staff somehow doesn't have a `is_admin` field with it, it will be considered admin by default.